snyk-monitor-deployment.yaml

```diff
@@ -19,107 +19,107 @@ spec:
         app.kubernetes.io/name: snyk-monitor
     spec:
       containers:
-      - image: snyk/kubernetes-monitor:IMAGE_TAG_OVERRIDE_WHEN_PUBLISHING
-        imagePullPolicy: Always
-        name: snyk-monitor
-        terminationMessagePath: /dev/termination-log
-        terminationMessagePolicy: File
-        volumeMounts:
+        - image: snyk/kubernetes-monitor:IMAGE_TAG_OVERRIDE_WHEN_PUBLISHING
```

Comment:
My formatter is doing something weird here 😕

Reply:
should you fix the indentation ?